### ISSUE TITLE
Github and Twitter logos in XAML

### ISSUE THREAD

**Issue by ThunderFrame** [rubberduck-vba/Rubberduck#1125]
@Hosch250 I can do one of these for twitter too...

```xaml
<!-- Created with Ai->XAML Export Plug-In Version 0.2 (PC/64) -->
<!-- By Mike Swanson (http://blog.mikeswanson.com/)           -->

<Viewbox Width="120.000" Height="117.037"
  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
<Canvas Width="120.000" Height="117.037">

  <Canvas>

  <!-- Layer 1/<Path> -->
  <Path Fill="#ff171515" Data=" M 59.996,0.000 C 26.866,0.000 0.000,26.863 0.000,60.003 C 0.000,86.510 17.190,108.999 41.035,116.934 C 44.035,117.484 45.130,115.632 45.130,114.041 C 45.130,112.617 45.078,108.844 45.049,103.837 C 28.359,107.463 24.837,95.794 24.837,95.794 C 22.109,88.862 18.175,87.017 18.175,87.017 C 12.728,83.294 18.588,83.370 18.588,83.370 C 24.609,83.794 27.778,89.553 27.778,89.553 C 33.130,98.722 41.821,96.074 45.240,94.539 C 45.785,90.661 47.335,88.017 49.049,86.517 C 35.725,85.003 21.719,79.856 21.719,56.863 C 21.719,50.313 24.057,44.957 27.895,40.763 C 27.278,39.245 25.219,33.145 28.483,24.884 C 28.483,24.884 33.521,23.271 44.982,31.035 C 49.768,29.704 54.901,29.039 60.003,29.016 C 65.099,29.039 70.232,29.704 75.025,31.035 C 86.479,23.271 91.508,24.884 91.508,24.884 C 94.781,33.145 92.722,39.245 92.105,40.763 C 95.950,44.957 98.272,50.313 98.272,56.863 C 98.272,79.913 84.244,84.985 70.877,86.471 C 73.032,88.323 74.951,91.985 74.951,97.582 C 74.951,105.603 74.877,112.074 74.877,114.041 C 74.877,115.646 75.958,117.513 79.003,116.927 C 102.824,108.977 120.000,86.503 120.000,60.003 C 120.000,26.863 93.134,0.000 59.996,0.000 Z"/>

  </Canvas>
</Canvas>
</Viewbox>
```

**Comment by ThunderFrame** [comment]
And twitter...

```xaml
<!-- Created with Ai->XAML Export Plug-In Version 0.2 (PC/64) -->
<!-- By Mike Swanson (http://blog.mikeswanson.com/)           -->

<Viewbox Width="120.000" Height="120.000"
  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
<Canvas Width="120.000" Height="120.000">

  <Canvas>

  <!-- Layer 1/<Compound Path> -->
  <Path Fill="#ff221e1f" Data="F1 M 92.670,42.747 C 92.702,43.460 92.718,44.177 92.718,44.897 C 92.718,66.860 76.001,92.185 45.431,92.185 C 36.045,92.185 27.310,89.433 19.954,84.718 C 21.255,84.871 22.577,84.950 23.919,84.950 C 31.706,84.950 38.872,82.293 44.560,77.836 C 37.287,77.701 31.149,72.896 29.034,66.293 C 30.049,66.487 31.090,66.591 32.161,66.591 C 33.677,66.591 35.145,66.388 36.540,66.009 C 28.937,64.482 23.208,57.765 23.208,49.712 C 23.208,49.642 23.208,49.572 23.209,49.503 C 25.450,50.748 28.013,51.496 30.737,51.582 C 26.278,48.601 23.344,43.514 23.344,37.748 C 23.344,34.703 24.163,31.848 25.594,29.393 C 33.791,39.448 46.037,46.065 59.850,46.758 C 59.567,45.541 59.420,44.273 59.420,42.970 C 59.420,33.792 66.861,26.351 76.040,26.351 C 80.820,26.351 85.139,28.369 88.171,31.599 C 91.957,30.854 95.514,29.471 98.725,27.566 C 97.484,31.447 94.849,34.704 91.417,36.761 C 94.779,36.359 97.982,35.466 100.963,34.144 C 98.735,37.477 95.917,40.404 92.670,42.747 Z M 60.000,0.000 C 26.863,0.000 0.000,26.863 0.000,60.000 C 0.000,93.137 26.863,120.000 60.000,120.000 C 93.137,120.000 120.000,93.137 120.000,60.000 C 120.000,26.863 93.137,0.000 60.000,0.000 Z"/>

  </Canvas>
</Canvas>
</Viewbox>
```

**Comment by ThunderFrame** [comment]
Twitter at 35px
```xaml
<!-- Created with Ai->XAML Export Plug-In Version 0.2 (PC/64) -->
<!-- By Mike Swanson (http://blog.mikeswanson.com/)           -->

<Viewbox Width="35.000" Height="35.000"
  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
<Canvas Width="35.000" Height="35.000">

  <Canvas>

  <!-- Layer 1/<Compound Path> -->
  <Path Fill="#ff221e1f" Data="F1 M 27.029,12.468 C 27.038,12.676 27.043,12.885 27.043,13.095 C 27.043,19.501 22.167,26.887 13.251,26.887 C 10.513,26.887 7.965,26.085 5.820,24.709 C 6.199,24.754 6.585,24.777 6.976,24.777 C 9.247,24.777 11.338,24.002 12.997,22.702 C 10.875,22.663 9.085,21.261 8.468,19.336 C 8.764,19.392 9.068,19.422 9.380,19.422 C 9.822,19.422 10.251,19.363 10.657,19.253 C 8.440,18.807 6.769,16.848 6.769,14.499 C 6.769,14.479 6.769,14.459 6.769,14.438 C 7.423,14.801 8.170,15.020 8.965,15.045 C 7.664,14.175 6.809,12.692 6.809,11.010 C 6.809,10.122 7.048,9.289 7.465,8.573 C 9.856,11.506 13.428,13.436 17.456,13.638 C 17.374,13.283 17.331,12.913 17.331,12.533 C 17.331,9.856 19.501,7.686 22.178,7.686 C 23.572,7.686 24.832,8.274 25.717,9.216 C 26.821,8.999 27.858,8.596 28.795,8.040 C 28.433,9.172 27.664,10.122 26.663,10.722 C 27.644,10.605 28.578,10.344 29.447,9.959 C 28.798,10.931 27.976,11.785 27.029,12.468 Z M 17.500,0.000 C 7.835,0.000 0.000,7.835 0.000,17.500 C 0.000,27.165 7.835,35.000 17.500,35.000 C 27.165,35.000 35.000,27.165 35.000,17.500 C 35.000,7.835 27.165,0.000 17.500,0.000 Z"/>

  </Canvas>
</Canvas>
</Viewbox>
```

**Comment by Hosch250** [comment]
Thanks.  This will be closed with #1128
